go.mod

```diff
@@ -230,9 +230,15 @@ replace (
 	// Use special SDK v0.47.x release with support for both ICS and LSM
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.47.13-ics-lsm
 
+	github.com/cosmos/gogoproto => github.com/cosmos/gogoproto v1.4.10
```

Comment:
These three new replacement directives are due to the feemarket module bumping this dependencies, after which Gaia stopped compiling, so we had to revert to previous versions.